garage/logger/snapshotter.py

```diff
@@ -78,3 +84,44 @@ def save_itr_params(self, itr, params):
             if file_name:
                 with open(file_name, 'wb') as file:
                     pickle.dump(params, file)
+
+    def load(self, itr='last'):
+        r"""Load one snapshot of parameters from disk.
+
+        Args:
+            itr(int or string): Iteration to load.
+                Can be an integer, 'last' or 'first'.
+
+        Returns:
+            dict: Loaded snapshot
+
+        """
+        if itr is None:
```

Comment:
why would someone pass `None` (when `'last'` is already default), and expect this to work?

Reply:
Fixed.